salt/modules/htpasswd.py

```diff
@@ -139,3 +139,46 @@ def userdel(pwfile, user, runas=None):
     out = __salt__['cmd.run'](cmd, runas=runas,
                               python_shell=False).splitlines()
     return out
+
+
+def verify(pwfile, user, password, opts='', runas=None):
+    '''
+    Return True if the htpasswd file exists, the user has an entry, and their
+    password matches.
+
+    pwfile
+        Path to htpasswd file
+
+    user
+        User name
+
+    password
+        User password
+
+    opts
+        Valid options that can be passed are:
```

Comment:
@heewa Thanks for mentioning that. I didn't think that you dismissed me. I've just been busy with some other things and I wanted to wait until I can give this my full attention. I'll get to it here shortly. Many thanks for your patience. :]